data-prepper-plugins/geoip-processor/src/main/java/org/opensearch/dataprepper/plugins/geoip/processor/GeoIPProcessorConfig.java

```diff
@@ -32,19 +34,31 @@ public class GeoIPProcessorConfig {
 
     @JsonProperty("tags_on_engine_failure")
     @JsonPropertyDescription("The tags to add to the event metadata if the <code>geoip</code> processor is unable to enrich an event due to an engine failure.")
+    @ExampleValues({
```

Comment:
Somewhat similar to example values on keys, I'm not sure that sample tag names offer much value. These are just identifiers. Thoughts?